database/database.go

```diff
@@ -301,6 +360,87 @@ func (db *Database) Close() {
  * Private
  **********************************************************************************************************************/
 
+func (db *Database) getDataFromQuery(query string, queryParams []any, result ...interface{}) error {
+	stmt, err := db.sql.Prepare(query)
+	if err != nil {
+		return aoserrors.Wrap(err)
+	}
+	defer stmt.Close()
+
+	if err = stmt.QueryRow(queryParams...).Scan(result...); err != nil {
+		if errors.Is(err, sql.ErrNoRows) {
+			return errNotExist
+		}
+
+		return aoserrors.Wrap(err)
+	}
+
+	return nil
+}
+
+func (db *Database) getDownloadFromQuery(
```

Comment:
What is the purpose of this function? Can be inside GetDownloadInfos.

Reply:
Remove